test/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetDrawer_Test.ts

```diff
@@ -9,6 +9,7 @@ import {IXmlElement} from "../../../../src/Common/FileIO/Xml";
 import {Fraction} from "../../../../src/Common/DataObjects/Fraction";
 import {VexFlowBackend} from "../../../../src/MusicalScore/Graphical/VexFlow/VexFlowBackend";
 import {CanvasVexFlowBackend} from "../../../../src/MusicalScore/Graphical/VexFlow/CanvasVexFlowBackend";
+import { DrawingParameters } from "../../../../src/MusicalScore/Graphical/DrawingParameters";
```

Comment:
Maybe no spaces here. I like it better with the spaces but we should be consistent I guess.